src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -139,7 +139,7 @@ export default {
 		t,
 
 		handleEnter(event) {
-			event.target.checked = !event.target.checked
+			this.modelProxy = !event.target.checked
```

Comment:
`modelProxy` is a proxy to `checked` which is an array, while `!event.target.checked` is a boolean

Reply:
`event.target.click()` may emulate native behavior. Just in case I'd prevent the default behavior in this case (but tested that in no browser Enter works as Space)

Reply:
Didn't check both cases, so came to a wrong assumption 🙈 

Initial goal was to emit `click-participant` event, which:
- doesn't require changing the `input.target.checked`:
- isn't reflected in UI (not a bulk selection => works almost as a button)